src/storage/v2/inmemory/label_index.cpp

```diff
@@ -235,4 +235,13 @@ std::vector<LabelId> InMemoryLabelIndex::Analysis() const {
   }
   return res;
 }
+
+void InMemoryLabelIndex::DropGraphClearIndices() {
+  for (auto &[label, storage] : index_) {
+    storage.clear();
+  }
+  index_.clear();
```

Comment:
`std::map<>::clear()` will destruct all the elements it is holding, since the `SkipList` destructor cleans itself up completely, there is no need to explicitly call `storage.clear()` on every skip-list here.
The function call on line 243 should be enough.